### PR TITLE
No CIDs for NATs, addresses #87

### PR DESCRIPTION
**Pull request by britram**

### CODE REVIEW

File: draft-ietf-quic-manageability.md [hunk 1]
```diff
@@ -696,7 +696,11 @@ requirements.
 The QUIC header optionally contains a Connection ID which can be used as
 additional entropy beyond the 5-tuple, if needed. The QUIC handshake needs
 to be observed in order to understand whether the Connection ID is present and
-what length it has.
+what length it has. However, Connection IDs may be renegotiated during
```
Comment (nibanks) [inline draft-ietf-quic-manageability.md]: Perhaps it should be explicitly mentioned that even the length of the CIDs can change. A change in length could break any logic that observes a handshake and assumes all CIDs have the same length.

Not to mention that the tuple itself could be shared by several parallel connections, each with different CIDs (possibly with different lengths).

**Comment by martinduke** [comment]
At Gorry's suggestion I put the NAT guidance into a different draft:
https://datatracker.ietf.org/doc/draft-duke-quic-natsupp/

Sorry, I thought the notification went out automatically. Regardless, this change is fine.

File: draft-ietf-quic-manageability.md [hunk 2]
```diff
@@ -696,7 +696,11 @@ requirements.
...
+a connection, and this renegotiation is not visible to the path. Keying state 
+off the Connection ID may therefore cause undetectable and unrecoverable loss 
+of state in the middle of a connection. Use of Connection ID specifically
```
Comment (ianswett) [inline draft-ietf-quic-manageability.md]:
```suggestion
of state in the middle of a connection. Use of Connection ID is specifically 
```

Comment (ianswett) [reply draft-ietf-quic-manageability.md]: Or maybe you want the is below, but I think something is missing?

**Pull request merged**